### ISSUE TITLE
Change router's `trailing_slash` option

### ISSUE THREAD

**Issue by spac3-monkey** [Tivix/django-rest-auth#258]
Hi,
Is there a way to change the `trailing_slash` option for the included rest auth URLs?

Thanks

**Comment by awwester** [comment]
I'm needing this same thing. Looking at the source code there doesn't seem to be any good way to get rid of the trailing slash without adding in your own urls.

**Comment by awwester** [comment]
I'm surprised there haven't been any issues opened about this. Is this something that I could make a PR for or are we expected just to override the urlpatterns manually?

**Comment by mariodev** [comment]
@awwester @spac3-monkey Can you give us an example case where existing trailing slash is causing a problem?

**Comment by awwester** [comment]
@mariodev it's for the same reason Django Rest Framework has `trailing_slash` option on their routers - when the client app wants to issue requests without the trailing slash and needs 2xx responses instead of a 301 redirect

**Comment by mariodev** [comment]
Seems to me like this could be resolved on the frontend side, by either attaching trailing slash or using a hack (e.g. for angular: https://github.com/angular-ui/ui-router/wiki/Frequently-Asked-Questions#how-to-make-a-trailing-slash-optional-for-all-routes)

However, please feel free to provide a PR if you feel strongly about the backend side solution. Thanks!

**Comment by awwester** [comment]
Yeah there are definitely multiple workarounds on the frontend or the backend. In most all of the projects I've worked on the frontend devs tend to want URLs without trailing slashes (it's kind of the norm outside of the Django world). I find it cleaner to just override the auth urls and remove the trailing slash rather than tell them to use trailing slashes.

Also it's in Django Rest Framework, and with this being an extension of DRF it would be nice to support trailing slashes in a similar fashion. I'll be putting together a PR for an optional setting and see what the feedback is. Thanks!

**Comment by kevinmanncito** [comment]
I don't even know that it should be an optional setting. What would be the issue with just having the urls always work with or without a slash? For my DRF endpoints I almost always just do this:

`r'^endpoint/?$'` instead of the typical: `r'^endpoint/$'`

Does anyone see any issues with making this change to the url confs?
